### PR TITLE
fixed bugs, ready for release

### PR DESCRIPTION
**Pull request by msokoloff1**
* Some extra information from the mal validation process was being added to the upload. This caused mal for subclasses to fail. 
* Bumped the version and updated the changelog so we are ready to release a fix.
* Updated the README to include that the SDK works for python3.6 
* Fixed an incorrect reference to a notebook in the examples README.

### CODE REVIEW

File: labelbox/schema/bulk_import_request.py [hunk 1]
```diff
@@ -633,13 +633,12 @@ def validate_subclasses(cls, value, field):
         #This is caused by the fact that we require these ids for top level classifications but not for subclasses
         results = []
         for row in value:
-            row.update({
-                'dataRow': {
+            results.append({
+                **row, 'dataRow': {
                     'id': 'child'.center(25, '_')
```
Comment (nmaswood) [inline labelbox/schema/bulk_import_request.py:638]: I would maybe move `'child`.center(25, '_')` out of the loop, it makes it more clear that it's just a dummy fill in value.

**Review by nmaswood — APPROVED** [review]

File: labelbox/schema/bulk_import_request.py (hunk 1 above)
Comment (msokoloff1) [reply labelbox/schema/bulk_import_request.py:638]: Good call!

**Review by nmaswood — APPROVED** [review]
if you like you could even put at module scope

```python
DUMMY_ID  =...
```

File: labelbox/schema/bulk_import_request.py (hunk 1 above)
Comment (msokoloff1) [reply labelbox/schema/bulk_import_request.py]: Fixed.

**Pull request merged**